README.md

```diff
@@ -73,17 +73,33 @@ for the docker worker in particular these are important:
 
 ## Running tests
 
-The `./test/test.sh` script is used to run individual tests which are
-suffixed with `_test.js` for example: `./test/test.sh test/integration/live_log_test.js`.
+There are a few components that must be configured for the tests to work properly (e.g. docker, kernel
```

Comment:
Yea, if one wants to run within docker on their linux host, they are more than capable of doing so.  They will also need some of the kernel modules for video and sound devices that we use if they want the loopback devices to work, which I've seen some people not have configured.